aten/src/ATen/native/quantized/README.md

```diff
@@ -7,24 +7,24 @@ This document serves as an entry point for quantized kernel implementation.
 ## Implementing native quantized ops
 
 The new quantized ops are almost always located under the `ATen/native/quantized/cpu` folder.
-For the sake of an example, let us implement an element-wise quantized logical AND operation under `ATen/native/quantized/cpu/qand.cpp`.
+For the sake of an example, let us implement an element-wise quantized logical XAND operation under `ATen/native/quantized/cpu/qxand.cpp`.
```

Comment:
what is "xand"? is this related to the rest of the PR?

Reply:
This is to replace an example in quantized outmoded by this patch. AND has already been implemented, so I replaced it with XAND.

Reply:
does 'xand' mean anything or it's just a dummy op-name for an example? If it's just an example, maybe we can name it something clearer?

Reply:
Yes, [xand](https://en.wiktionary.org/wiki/XAND) has its own meaning

Reply:
that seems a bit confusing to me, but I guess that's OK

Reply:
I've added a hyperlink to an explanation of xand